src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -32,15 +39,29 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = (props) => {
 
   const { patientName, patientID, patientBirthdate, patientGender } = createAnalysis;
   const { submitAnalysis } = props;
-  
+
+  const handleXrayChange = (model: string) => {
+    setXrayModel(model);
+  }
+
+  const handleCTChange = (model: string) => {
+    setCTModel(model);
+  }
+
+  const setModelType = (index: number) => {
+    if (index !== modelValue) { // Checking that clicked image tab is not current image tab
+      setUseXray(!useXray);
+      setModelValue(index);
+    }
+  }
```

Comment:
Continuing from the comment above, `setModelType` could be modified to something like this:
```suggestion
  const setModelType = (modality: string) => {
    setModelValue(modality);
    setUseXray(modality === 'CR');
  }
```